AirportTicketBooking/Models/Booking.cs

```diff
@@ -1,14 +1,17 @@
-﻿using System;
-
-namespace AirportTicketBooking.Models;
+﻿namespace AirportTicketBooking.Models;
 
 public class Booking
 {
-    public required int Id { get; init; }
-    public required int FlightId{ get; init; }
-    public required int PassengerId { get; init; }
+    public string Id { get; set; }
+    public string FlightId { get; set; }
+    public string PassengerId { get; set; }
 
-    public Booking(int id, int flightId, int passengerId)
+    public Booking(string flightId, string passengerId)
+    {
+        FlightId = flightId;
+        PassengerId = passengerId;
+    }
```

Comment:
Needs to set initial value for missing property.
```suggestion
    public Booking(string flightId, string passengerId)
    {
        Id = string.Empty;
        FlightId = flightId;
        PassengerId = passengerId;
    }
```